src/App.tsx

```diff
@@ -12,155 +15,44 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            to="/"
+            className={({ isActive }) => {
+              return cn('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              });
+            }}
+          >
             Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          </NavLink>
+
+          <NavLink
+            to="/people"
+            className={({ isActive }) => {
+              return cn('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              });
```

Comment:
I think it is better to make a separate function, so you can improve the readability of the code

Reply:
Good point!